index.html

```diff
@@ -366,12 +368,15 @@
 &lt;script&gt;
   // read presId from localStorage if exists
   var presId = localStorage &amp;&amp; localStorage["presId"] || null;
-  // presId is mandatory for joinSession.
-  presId &amp;&amp; navigator.presentation.joinSession(presUrl, presId)
-    // The joined session will be passed to setSession on success
-    .then(setSession)
-    // no session found for presUrl and presId or an error is occurred
-    .catch(endSession);
+  // presId is mandatory when joining a session.
+  if (presId) {
+    var request = new PresentationRequest(presUrl);
```

Comment:
I think in the examples, we are reusing all the definitions from the previous ones, so there's no need to create a new PresentationRequest in each example. It might confuse developers, who will create a separate request availability, starting and joining presentation, etc.